### PR TITLE
docs(plasma-ui, plasma-web, plasma-temple): change links for new site

### PR DESCRIPTION
**Pull request by kayman233**
:baby_chick: Download canary assets:

[colors--canary.238.3571905144.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.238.3571905144.xml)  
[color_sbermarket_ios-swift--canary.238.3571905144.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.238.3571905144.swift)  
[color_sbermarket_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.238.3571905144.kt)  
[color_sbermarket_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.238.3571905144.ts)  
[color_sberprime_ios-swift--canary.238.3571905144.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.238.3571905144.swift)  
[color_sberprime_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.238.3571905144.kt)  
[color_sberprime_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.238.3571905144.ts)  
[PlasmaTokensColor--canary.238.3571905144.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.238.3571905144.swift)  
[typo_mage_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.238.3571905144.kt)  
[typo_mage_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.238.3571905144.ts)  
[typo_plasma_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.238.3571905144.kt)  
[typo_plasma_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.238.3571905144.ts)  
[typo_ruler_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.238.3571905144.kt)  
[typo_ruler_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.238.3571905144.ts)  
[typo_sage_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.238.3571905144.kt)  
[typo_sage_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.238.3571905144.ts)  
[typo_sbermarket_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.238.3571905144.kt)  
[typo_sbermarket_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.238.3571905144.ts)  
[typo_soulmate_kotlin--canary.238.3571905144.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.238.3571905144.kt)  
[typo_soulmate_react-native--canary.238.3571905144.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.238.3571905144.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-238/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-238/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-238/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-238/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-238/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-238/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-238/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-238/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-238/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-238/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-238/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-238/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-238/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-238/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-238/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-238/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-238/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-238/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-238/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-238/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-238/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-238/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-238/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-238/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-238/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-238/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-238/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-238/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-238/temple-storybook/

File: website/plasma-website/components/main/Menu.tsx [hunk 1]
```diff
@@ -49,9 +61,7 @@ export const Menu: FC<MenuProps> = ({ items, style, className }) => (
             {items.map((item, i) => (
                 <Item key={`item:${i}`}>
                     {!item.soon ? (
-                        <Link href={item.href} external={item.external}>
-                            {item.title}
-                        </Link>
+                        <LinkTitle href={item.href}>{item.title}</LinkTitle>
```
Comment (Yeti-or) [inline website/plasma-website/components/main/Menu.tsx:64]: external не нужен больше ?

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**